jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/LoggingMXBeanImpl.java

```diff
@@ -49,8 +48,47 @@
 /*[ENDIF]*/
 			PlatformLoggingMXBean {
 
-	private static final LoggingMXBeanImpl instance = new LoggingMXBeanImpl();
-
+	private static final LoggingMXBeanImpl instance;
+	
+	private static final Class<?> logManagerClass;
+	private static final Class<?> loggerClass;
+	private static final Class<?> levelClass;
+	private static final Method lm_getLogManager;
+	private static final Method lm_getLogger;
+	private static final Method lm_getLoggerNames;
+	private static final Method lo_getLevel;
+	private static final Method lo_getParent;
+	private static final Method lo_getName;
+	private static final Method lo_setLevel;
+	private static final Method le_getName;
+	private static final Method le_parse;
+	private static final String lm_LOGGING_MXBEAN_NAME;
+	
+	static {
+		try {
+			logManagerClass = Class.forName("java.util.logging.LogManager");
+			loggerClass = Class.forName("java.util.logging.Logger");
+			levelClass = Class.forName("java.util.logging.Level");
+			lm_getLogManager = logManagerClass.getMethod("getLogManager");
+			lm_getLogger = logManagerClass.getMethod("getLogger", String.class);
+			lm_getLoggerNames = logManagerClass.getMethod("getLoggerNames");
+			lo_getLevel = loggerClass.getMethod("getLevel");
+			lo_getParent = loggerClass.getMethod("getParent");
+			lo_getName = loggerClass.getMethod("getName");
+			lo_setLevel = loggerClass.getMethod("setLevel", levelClass);
+			le_getName = levelClass.getMethod("getName");
+			le_parse = levelClass.getMethod("parse", String.class);
```

Comment:
lm = LogManager.
lo = logger?
le = level?

Can more explicit names be used?  It will be easier on readers when trying to figure out what class is being used.